packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx

```diff
@@ -54,16 +54,17 @@ export const usePresenceBadge_unstable = (
   props: PresenceBadgeProps,
   ref: React.Ref<HTMLElement>,
 ): PresenceBadgeState => {
+  const statusText = DEFAULT_STRINGS[props.status ?? 'available'];
+  const oofText = props.outOfOffice && props.status !== 'out-of-office' ? ` ${DEFAULT_STRINGS['out-of-office']}` : '';
   const state: PresenceBadgeState = {
     ...useBadge_unstable(
       {
         size: 'medium',
+        'aria-label': statusText + oofText,
```

Comment:
I am not sure if this will work in all languages - if we want to construct labels, we should probably expose callback that allows localization of these texts using a standardized i18n library.

Reply:
This already isn't localizable, so it isn't super relevant until we actually have strings that can be passed in.

That said, this particular one doesn't worry me too much, since it's not a grammatical connection, just two separate pieces of adjacent information.